packages/sui-pde/src/hooks/useDecisionCallback.js

```diff
@@ -0,0 +1,66 @@
+import {useCallback, useContext} from 'react'
+
+import PdeContext from '../contexts/PdeContext.js'
+import {getPlatformStrategy} from './common/platformStrategies.js'
+
+/**
+ * Hook to call decide
+ * @return {function}
+ */
+export default function useDecisionCallback() {
```

Comment:
This allows the `decide` function to be called programmatically, which is useful for very specific use cases. 99% `useDecision` will be just fine